chainio/clients/eigenpod/generate.sh

```diff
@@ -18,7 +18,7 @@ cleanup_bindings_dir() {
 
 clone() {
   echo "Cloning the EigenLayer contracts repository"
-  git clone -b slashing-magnitudes --depth=1 https://github.com/Layr-Labs/eigenlayer-contracts.git "${TMP_DIR}"
+  git clone --depth=1 https://github.com/Layr-Labs/eigenlayer-contracts.git "${TMP_DIR}"
```

Comment:
I'm not sure if this is what we want. We should probably use the same `eigenlayer-contract` ref we have under `contracts/` instead